src/nanoemoji/paint.py

```diff
@@ -115,17 +118,25 @@ def breadth_first(self) -> Generator[PaintTraverseContext, None, None]:
     def children(self) -> Iterable["Paint"]:
         return ()
 
+    def gettransform(self) -> Optional[Affine2D]:
+        # Returns the transform caused by this Paint (not it's ancestors)
+        return Affine2D.identity()
+
 
 @dataclasses.dataclass(frozen=True)
 class PaintColrLayers(Paint):
     format: ClassVar[int] = int(ot.PaintFormat.PaintColrLayers)
     layers: Tuple[Paint, ...]
 
     def colors(self):
-        pass
+        for p in self.layers:
+            yield from p.colors()
 
     def to_ufo_paint(self, colors):
-        return [p.to_ufo_paint(colors) for p in self.layers]
+        return {
+            "Format": self.format,
+            "Layers": [p.to_ufo_paint(colors) for p in self.layers],
```

Comment:
why are you returning a dict instead of a list? I think colorLib.builder already special-case list of paints as meaning PaintColrLayers,  no? The plist encoded in the UFO.lib will be a bit more compact if you use a list (encoded as plist `<array>` as opposed to a `<dict>` with all the key/values and the implied Format.

Reply:
I liked the consistency of always using the fieldname: value dict.

Reply:
That said, it's just pointlessly making this change larger so I think I'll revert to the old form